core/inspector.rs

```diff
@@ -492,7 +506,7 @@ impl task::ArcWake for InspectorWaker {
 /// eg. Websocket or another set of channels.
 struct InspectorSession {
   v8_channel: v8::inspector::ChannelBase,
-  v8_session: Rc<RefCell<v8::UniqueRef<v8::inspector::V8InspectorSession>>>,
+  v8_session: NonNull<v8::inspector::V8InspectorSession>,
```

Comment:
You need to keep using `UniqueRef` here, otherwise this will cause a memory leak (since you're using `UniqueRef::into_raw()` below).